pkg/storaged/content-views.jsx

```diff
@@ -494,6 +494,8 @@
         type = C_("storage-id-desc", "VDO backing");
         used_for = vdo.name;
         link = ["vdo", vdo.name];
+    } else if (client.blocks_swap[block.path]) {
+        type = C_("storage-id-desc", "Swap space");
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17929-20221117-190140-3f947163-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)